ospo-book/chapters/03-chapter.md

```diff
@@ -1 +1,146 @@
+In this chapter, we will introduce the key tasks and responsibilities of an open source program office (OSPO) on a day-to-day basis, 
+as well as the core concepts to get started with a minimum viable OSPO. This includes the strategy aspect, layers of work, and pillars of support.
+
+### Ingredients for a Minimum Viable OSPO
+
+The first step in establishing an OSPO is to shape and build up the structure. This includes personnel, technology, and budget. It is also 
+important to identify the key stakeholders in your organization who will be involved in open source activities.
+
+The guide [A deep dive into OSPO](https://www.linuxfoundation.org/research/a-deep-dive-into-open-source-program-offices) explains all essential 
+information on OSPO structures and operations.
+
+Once this is evaluated, it is recommended to look at the OSPO from three different dimensions:
+
+* The layers where the OSPO works and interacts
+* The pillars where the OSPO provides support
+* The strategy it will follow and its framework
+
+### Layers of work for an OSPO
+
+The area of work of your OSPO is also a crucial component to map, as it serves as one of the core ingredients for effectively implementing open source 
+dynamics and philosophy on an organizational level. To achieve this, it is recommended to examine your own organizational architecture and break down 
+open source tasks into four layers of work:
+
+![ospo-layers](https://github.com/todogroup/ospology/assets/43671777/03b224d5-9745-4581-ac36-0dbbda9a990d)
+
+* **Worker / Employee Level:** This layer involves the people behind the OSPO to wor wiht individuals within the organization, wiht main focus on the intrinsic and extrinsic motivators of contributing 
+ to open source from an individual point of view. It requires a cultural change effort and may involve activities such as establishing mentoring programs.
+
+* **Management Level:** At this layer, the people behind the OSPO focus is on strategy and finding alignment between open source and the overall business / organizaiton strategy. Managers may face unique challenges, and leveraging the strengths of open source to help them solve these challenges can help address these issues effectively.
+
+* **Business / Team Units Level:** This layer focuses on how the people behind the OSPO ensures all the pieces of the organization structure fit together. It involves sharing best practices across different business / Team units and fostering collaboration and knowledge transfer among them.
+
+* **OSPO Level:** This is the inner side of the OSPO itself, where the people behind the OSPO works collectively to oversee and coordinate all the open source initiatives within the organization.
+
+This is a simplified view of the layers of work in an OSPO. However, depending on the complexity of your organization and the resources available to your OSPO, these layers can become more granular and include additional layers with different names.
+
+### The Pillars of Support
```

Comment:
This overlaps strongly with the roles defined in Chapter 2.

Reply:
I think you are right @gkunz. If everyone is ok with that, I will delete this section and include the word "pillars of support in the chapter 2 dedicated section so we don't duplicate content.